pkg/publish/ingress.go

```diff
@@ -19,12 +19,12 @@ import (
 )
 
 func toPrefix(serviceName string, appInstance *v1.AppInstance) string {
-	hostPrefix := serviceName + "." + appInstance.Name
+	hostPrefix := serviceName + "---" + appInstance.Name
 	if serviceName == "default" {
 		hostPrefix = appInstance.Name
 	}
 	if appInstance.Namespace != system.DefaultUserNamespace {
-		hostPrefix += "." + appInstance.Namespace
+		hostPrefix += "---" + appInstance.Namespace
```

Comment:
What is this for?

Reply:
We can't use dots here since the wildcard cert is only valid one level deep and the triple dash is mostly to avoid ambiguities as mentioned by Vincent here: https://github.com/acorn-io/acorn/issues/205#issuecomment-1240070940 (does that make sense?)

Reply:
Whoops, forgot to add the `<appname>` part. Done.
Example: `http://my-webapp-dawn-moon-9b980585.v4avai.staging.on-acorn.io/`